modAionImpl/src/org/aion/zero/impl/blockchain/AionBlockchainImpl.java

```diff
@@ -236,6 +236,11 @@ protected AionBlockchainImpl(
             this.beaconHashValidator = new BeaconHashValidator(this,
                     BeaconHashValidator.FORK_050_DISABLED);
         } else {
+            if (maybeFork050.get() < 2) {   // AKI-419, Constrain the minimum unity fork number
+                LOG.info("The unity fork number can not less than 2, set the fork number to 2");
```

Comment:
I would log this under `log.warn`, also I suggest the following message rewrite: "The unity fork number is set to an invalid value (less than 2). Replacing incorrect configuration with a unity fork number equal to 2."